<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>672</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestionaire de stock</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/ressources/icons/truck.svg</normaloff>:/ressources/icons/truck.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedPages">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="settingsPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Param√©tres</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="1">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QFrame" name="frame_3">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QPushButton" name="buttonEditProducts">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Modifier les produits</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="homePage">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>StockActuel</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableActualStock">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="tabKeyNavigation">
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Item</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Qte</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Prix</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date d'achat</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date d'expiation</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Client</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Details</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_7">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>250</height>
              </size>
             </property>
             <property name="title">
              <string>Logo</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_13">
              <item row="0" column="0">
               <widget class="QLabel" name="labelLogo">
                <property name="minimumSize">
                 <size>
                  <width>220</width>
                  <height>220</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="inventoryPage">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Actions</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QPushButton" name="invBuy">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>2</width>
               <height>2</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>false</bold>
               <strikeout>false</strikeout>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>Achat</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/ressources/icons/box-arrow-down.svg</normaloff>:/ressources/icons/box-arrow-down.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="invSell">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>2</width>
               <height>2</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>false</bold>
               <strikeout>false</strikeout>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>Vente</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/ressources/icons/box-arrow-up.svg</normaloff>:/ressources/icons/box-arrow-up.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="invWasteBasket">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>2</width>
               <height>2</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>false</bold>
               <strikeout>false</strikeout>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>Corbeille</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/ressources/icons/trash.svg</normaloff>:/ressources/icons/trash.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="invReproduction">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>2</width>
               <height>2</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>false</bold>
               <strikeout>false</strikeout>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>Reproduction</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/ressources/icons/recycle.svg</normaloff>:/ressources/icons/recycle.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Historique</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableHistory">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="tabKeyNavigation">
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>110</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Action</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Item</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Qte</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Prix</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date d'expiration</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Lot</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Client</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Logo</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDashboard"/>
   <addaction name="actionInventory"/>
   <addaction name="actionParameters"/>
  </widget>
  <action name="actionDashboard">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ressources/icons/house.svg</normaloff>:/ressources/icons/house.svg</iconset>
   </property>
   <property name="text">
    <string>Tableau de Bord</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionInventory">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ressources/icons/box-seam.svg</normaloff>:/ressources/icons/box-seam.svg</iconset>
   </property>
   <property name="text">
    <string>Inventaire</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionParameters">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ressources/icons/gear.svg</normaloff>:/ressources/icons/gear.svg</iconset>
   </property>
   <property name="text">
    <string>Param√©tres</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="develop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ressources/icons/caret-right.svg</normaloff>:/ressources/icons/caret-right.svg</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
